src/Analyzers/CSharp/Analyzers/UseNullPropagation/CSharpUseNullPropagationDiagnosticAnalyzer.cs

```diff
@@ -31,8 +33,8 @@ protected override bool ShouldAnalyze(ParseOptions options)
         protected override ISyntaxFacts GetSyntaxFacts()
             => CSharpSyntaxFacts.Instance;
 
-        protected override ISemanticFactsService GetSemanticFactsService()
```

Comment:
I am planning to move `ISemanticFactsService` down to the shared analyzer layer in a follow-up PR. This work is also tracked in the follow-up issue. I will make sure to revert this change as part of that PR.